backend/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+authors = ["Mark-Simulacrum <mark.simulacrum@gmail.com>"]
+name = "rs-backend"
+version = "0.1.0"
+
+[dependencies]
+chrono = "0.2.22"
+error-chain = "0.2.2"
+json = "0.10.1"
+iron = "0.4.0"
+router = "0.2.0"
+persistent = "0.2.0"
+
+[profile.release]
+debug = true
```

Comment:
Should still get all the errors without compiling in debug mode, this just means we'll get better info in stack traces, etc. I would hope we don't need that and we can add this if needed later. OTOH, I guess we can leave it in as long as we remember to take it out in the future.